Jointify/Views/Subviews/DefaultButton.swift

```diff
@@ -46,3 +48,9 @@ struct DefaultButton_Previews: PreviewProvider {
     }
     
 }
+
```

Comment:
```suggestion

// MARK: Extension Color
/// add the default colour for the button
```

Reply:
We have a style guide which I persistantly push and will send you just now :-). The MARKs are just a hygiene thing.